samples/cpp/tutorial_code/HighGUI/AddingImagesTrackbar.cpp

```diff
@@ -6,9 +6,10 @@
 
 #include "opencv2/imgcodecs.hpp"
 #include "opencv2/highgui.hpp"
-#include <stdio.h>
+#include <cstdio>
 
 using namespace cv;
+using namespace std;
```

Comment:
Please avoid "using namespace std;". Use `std::cout` directly.
Especially in this file due "std::beta()" conflict (see line 33).